libpromises/evalfunction.c

```diff
@@ -2167,6 +2167,7 @@ static FnCallResult FnCallSplayClass(EvalContext *ctx,
         snprintf(class_name, CF_MAXVARSIZE, "Min%02d_%02d.Hr%02d", slot * 5, ((slot + 1) * 5) % 60, hour);
     }
 
+    Log(LOG_LEVEL_DEBUG, "Computed context for splayclass: '%s'", class_name);
```

Comment:
I think this should also log the ``input`` argument (``finalargs`` here) for easy identification.